molecule/mtls-ubuntu/molecule.yml

```diff
@@ -121,3 +121,5 @@ provisioner:
       all:
         ssl_enabled: true
         ssl_mutual_auth_enabled: true
+        secrets_protection_enabled: true # To test secrets protection on cli v3 without rbac
+        confluent_cli_version: 3.2.1
```

Comment:
Do you want to keep it to 3.2 even if we migrate the default cli version to higher versions ?

Reply:
the default cli version is 2.x for 7.1, I used 3.2 since we use that as the default cli 3 version in 7.4, though higher versions are also available